README.md

```diff
@@ -356,6 +356,17 @@ test-deploy-test cycle after this (the test after the deploy is required to test
         # do stuff     
     ```
 
+### Audit Logs
+The DSS uses AWS Cloudtrail and S3 to manage audit logs. By default audit logs are disable, but can be abled by setting 
+`ENABLE_AUDIT_LOGS` to 0 in the [environment file](environment) and providing a name for `DSS_AUDIT_LOGS_BUCKET`. The 
+Cloudtrail and S3 bucket will then be created by re-running `make deploy-infra`. Do not enable audit logs until after 
+you have sucesfully deployed the DSS at least once otherwise it will fail. This is because not all of the resources 
```

Comment:
`Do not enable audit logs until after you have successfully deployed the DSS at least once otherwise it will fail.`

I feel like this should be fixed now.  If not, and it should have an associated issue and warnings should be there wherever this is mentioned.